src/app/builders/components/Container/PageTitleContainer.tsx

```diff
@@ -0,0 +1,49 @@
+import { cn } from '@/lib/utils'
+import { FC } from 'react'
+import { CommonComponentProps } from '../commonProps'
+import { H1 } from '../H/H1'
+
+export type PageTitleContainerProps =
+  | (CommonComponentProps & {
+      children: React.ReactNode
+      leftText?: never
+      rightText?: never
+    })
+  | (CommonComponentProps & {
+      children?: never
+      leftText: string
+      rightText?: string
+    })
+
+const ContainerCompnent: FC<PageTitleContainerProps> = ({ dataTestid, className = '', children }) => (
+  <div data-testid={`${dataTestid}_PageTitleContainer`} className={cn('flex w-full', className)}>
+    {children}
+  </div>
+)
+
+export const PageTitleContainer: FC<PageTitleContainerProps> = ({
```

Comment:
This component is a little bit different from the Backers, but I would analyze if it can be customizable for both cases.